### PR TITLE
[KOGITO-9095][KSW-Guides] Make Workflow Dev Profile builder image configurable via Platform

### PR DESCRIPTION
**Pull request by desmax74**
**JIRA:** https://issues.redhat.com/browse/KOGITO-8675

**Description:**

Please make sure that your PR meets the following requirements:

- [ ] You have read the [contributions doc](https://github.com/kiegroup/kogito-docs/blob/main/CONTRIBUTING.md)
- [ ] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [ ] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [ ] The nav.adoc file has a link to this guide in the proper category
- [ ] The index.adoc file has a card to this guide in the proper category, with a meaningful description

<details>
<summary>
How to backport a pull request to a different branch?
</summary>

In order to automatically create a **backporting pull request** please add one or more labels having the following format `backport-<branch-name>`, where `<branch-name>` is the name of the branch where the pull request must be backported to (e.g., `backport-7.67.x` to backport the original PR to the `7.67.x` branch).

> **NOTE**: **backporting** is an action aiming to move a change (usually a commit) from a branch (usually the main one) to another one, which is generally referring to a still maintained release branch. Keeping it simple: it is about to move a specific change or a set of them from one branch to another.

Once the original pull request is successfully merged, the automated action will create one backporting pull request per each label (with the previous format) that has been added.

If something goes wrong, the author will be notified and at this point a manual backporting is needed.

> **NOTE**: this automated backporting is triggered whenever a pull request on `main` branch is labeled or closed, but both conditions must be satisfied to get the new PR created.
</details>

### CODE REVIEW

File: serverlessworkflow/modules/ROOT/pages/cloud/operator/workflow-dev-profile-builder-configurable.adoc [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+:description: Configuration workflow Dev Profile builder image via Platform
+:keywords: kogito, workflow, serverless, operator, kubernetes, minikube, config, openshift, containers
+
+This document describes how to configure Workflow Dev Profile builder image via Platform with the {operator_name}.
+
+#### Override Builder image and version
+
+By default the builder image use a stable version aligned to the Kogito version.
+
+In the dev mode you can override the default builder image using the devBaseImage in the KogitoServerlessPlatform:
```
Comment (kaldesai) [inline serverlessworkflow/modules/ROOT/pages/cloud/operator/workflow-dev-profile-builder-configurable.adoc:13]:
```suggestion
In the dev mode, you can override the default builder image using the devBaseImage in the KogitoServerlessPlatform:
```

**Review by kaldesai — APPROVED** [review]

File: serverlessworkflow/modules/ROOT/pages/cloud/index.adoc [hunk 2]
```diff
@@ -98,6 +98,16 @@ xref:cloud/operator/build-and-deploy-workflows.adoc[]
 Learn how to build and deploy workflow services with {operator_name}
 --
 
+[.card]
+--
+[.card-title]
+xref:cloud/operator/workflow-dev-profile-builder-configurable.adoc[]
```
Comment (spolti) [inline serverlessworkflow/modules/ROOT/pages/cloud/index.adoc:104]: can't we have this as part of the dev profile section instead add a new one?

Comment (ricardozanini) [reply serverlessworkflow/modules/ROOT/pages/cloud/index.adoc:104]: I agree. It's better to add a section to the Dev profile section.

**Review by ricardozanini — CHANGES_REQUESTED** [review]

File: serverlessworkflow/modules/ROOT/pages/cloud/index.adoc (hunk 2 above)
Comment (desmax74) [reply serverlessworkflow/modules/ROOT/pages/cloud/index.adoc:104]: Done

**Comment by desmax74** [comment]
@spolti is under this section https://github.com/kiegroup/kogito-docs/pull/338/files#diff-66eda6b7c91f919d69f1eff12caea9560ee132c7afa4d18106e5766e6bff3436R109 are you asking a separate folder and path ?

File: serverlessworkflow/modules/ROOT/pages/cloud/index.adoc [hunk 3]
```diff
@@ -105,3 +105,13 @@ xref:cloud/operator/known-issues.adoc[]
 [.card-description]
 Learn about the known issues and feature roadmap of the {operator_name}
 --
+
+== Operator Dev Mode
```
Comment (radtriste) [inline serverlessworkflow/modules/ROOT/pages/cloud/index.adoc:109]: Please also update nav.adoc

File: serverlessworkflow/modules/ROOT/pages/cloud/operator/workflow-dev-profile-builder-configurable.adoc [hunk 4]
```diff
@@ -0,0 +1,27 @@
+= Make Workflow Dev Profile builder image configurable via Platform
+:compat-mode!:
+// Metadata:
+:description: Configuration workflow Dev Profile builder image via Platform
+:keywords: kogito, workflow, serverless, operator, kubernetes, minikube, config, openshift, containers
+
+This document describes how to configure Workflow Dev Profile builder image via Platform with the {operator_name}.
+
+#### Override Builder image and version
```
Comment (radtriste) [inline serverlessworkflow/modules/ROOT/pages/cloud/operator/workflow-dev-profile-builder-configurable.adoc:9]:
```suggestion
== Override Builder image and version
```

File: serverlessworkflow/modules/ROOT/pages/cloud/operator/workflow-dev-profile-builder-configurable.adoc [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+By default the builder image use a stable version aligned to the Kogito version.
```
Comment (radtriste) [inline serverlessworkflow/modules/ROOT/pages/cloud/operator/workflow-dev-profile-builder-configurable.adoc:11]:
```suggestion
By default the builder image use a stable version aligned to the {product_name} version.
```

**Review by radtriste — CHANGES_REQUESTED** [review]

File: serverlessworkflow/modules/ROOT/pages/cloud/index.adoc (hunk 3 above)
Comment (desmax74) [reply serverlessworkflow/modules/ROOT/pages/cloud/index.adoc:109]: ok

File: serverlessworkflow/modules/ROOT/pages/cloud/operator/workflow-dev-profile-builder-configurable.adoc (hunk 4 above)
Comment (desmax74) [reply serverlessworkflow/modules/ROOT/pages/cloud/operator/workflow-dev-profile-builder-configurable.adoc:9]: ok

File: serverlessworkflow/modules/ROOT/pages/cloud/operator/workflow-dev-profile-builder-configurable.adoc (hunk 5 above)
Comment (desmax74) [reply serverlessworkflow/modules/ROOT/pages/cloud/operator/workflow-dev-profile-builder-configurable.adoc:11]: ok

File: serverlessworkflow/modules/ROOT/pages/cloud/operator/workflow-dev-profile-builder-configurable.adoc [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+:compat-mode!:
+// Metadata:
+:description: Configuration workflow Dev Profile builder image via Platform
+:keywords: kogito, workflow, serverless, operator, kubernetes, minikube, config, openshift, containers
+
+This document describes how to configure Workflow Dev Profile builder image via Platform with the {operator_name}.
+
+== Override Builder image and version
+
+By default the builder image use a stable version aligned to the {product_name} version.
```
Comment (MarianMacik) [inline serverlessworkflow/modules/ROOT/pages/cloud/operator/workflow-dev-profile-builder-configurable.adoc:11]:
```suggestion
By default the builder image uses a stable version aligned to the {product_name} version.
```

File: serverlessworkflow/modules/ROOT/pages/cloud/operator/workflow-dev-profile-builder-configurable.adoc [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+In the dev mode, you can override the default builder image using the devBaseImage in the KogitoServerlessPlatform:
```
Comment (MarianMacik) [inline serverlessworkflow/modules/ROOT/pages/cloud/operator/workflow-dev-profile-builder-configurable.adoc:13]:
```suggestion
In the dev mode, you can override the default builder image using the `devBaseImage` in the KogitoServerlessPlatform:
```

File: serverlessworkflow/modules/ROOT/pages/cloud/operator/workflow-dev-profile-builder-configurable.adoc (hunk 6 above)
Comment (desmax74) [reply serverlessworkflow/modules/ROOT/pages/cloud/operator/workflow-dev-profile-builder-configurable.adoc:11]: ok

File: serverlessworkflow/modules/ROOT/pages/cloud/operator/workflow-dev-profile-builder-configurable.adoc (hunk 7 above)
Comment (desmax74) [reply serverlessworkflow/modules/ROOT/pages/cloud/operator/workflow-dev-profile-builder-configurable.adoc:13]: ok

**Comment by ricardozanini** [comment]
Already done here: https://sonataflow.org/serverlessworkflow/latest/cloud/operator/developing-workflows.html

**Pull request closed without merging**